e2e/components/login/login.readonly.spec.js

```diff
@@ -56,11 +56,11 @@ describe('when logging in', () => {
         expect(component.toggleLoginComponent.getText()).toBe('AQA ONC STAFF');
     });
 
-    it('should be able to log in as developer with an email address', () => {
-        component.logInWithEmail('developer');
-        component.logoutButton.waitForDisplayed();
-        expect(component.toggleLoginComponent.getText()).toBe('AQA Developers');
-    });
+    // it('should be able to log in as developer with an email address', () => {
```

Comment:
Not sure if our system supports it for the AQA tests, but the unit tests in this feature can ignore a test by marking it as `xit` instead of `it`, and it will e`x`clude that test
```suggestion
    xit('should be able to log in as developer with an email address', () => {
```